swagger.yaml

```diff
@@ -4756,6 +4756,21 @@ components:
           type: string
           example: "6804edf9712d2b619edb6ac86861fe93a730693183a262b165fcc1ba1bc99cad"
           description: ID of the initial minting transaction
+        onchain_metadata:
+          type: object
+          nullable: true
```

Comment:
```suggestion
          type: object
          additionalProperties: true,
          nullable: true
```

Reply:
Needed if we want to support [arbitrary properties](https://github.com/cardano-foundation/CIPs/pull/85/files#diff-845f22e50f8ab4fbc50a67587aaf66ba2d18aa4c27c84d592ed08d22c8f570b7R57)